lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -238,7 +334,28 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     }
 
     Node *node = nullptr;
-    if (typeName == "MaxPool") {
+
+    if (typeName == "Int8MaxPool" || typeName == "Int8AveragePool") {
+      // Create the node with quantized type.
+      assert(dict.count("Y_zero_point") &&
+             "missing zero point for quantzied type");
+      assert(dict.count("Y_scale") && "missing Y_scale for quantzied type");
+      ShapeNHWC idim = ShapeNHWC(tr->getType(0)->dims());
```

Comment:
it would be less error prone to get result and type from the result and avoid explicit indexing?

Reply:
`tr` could either be the input node directly or a node with Transposed type. So I think it is easier to get the type directly without casting.